### PR TITLE
[airframes] RW25kg

### PR DESCRIPTION
**Pull request by dewagter**

### CODE REVIEW

File: conf/flight_plans/tudelft/rotating_wing25kg_EHVB.xml [hunk 1]
```diff
@@ -0,0 +1,168 @@
+<!DOCTYPE flight_plan SYSTEM "../flight_plan.dtd">
+
+<flight_plan alt="60" ground_alt="0" lat0="52.168595" lon0="4.412444" max_dist_from_home="1100" name="Rotating wing Valkenburg" security_height="2">
+  <header>
+    #include "autopilot.h"
+    #include "modules/datalink/datalink.h"
+</header>
```
Comment (gautierhattenberger) [inline conf/flight_plans/tudelft/rotating_wing25kg_EHVB.xml:7]: I don't think these includes are needed anymore. All headers included by modules are available here as well.

File: conf/flight_plans/tudelft/rotating_wing25kg_EHVB.xml [hunk 2]
```diff
@@ -0,0 +1,168 @@
...
+      <attitude pitch="0" roll="0" throttle="0" until="FALSE" vmode="throttle"/>
+    </block>
+    <block name="Start Engine">
+        <set value="false" var="force_forward"/>
+        <call_once fun="NavResurrect()"/>
+        <attitude pitch="0" roll="0" throttle="0" until="stage_time>10" vmode="throttle"/>
+    </block>
+    <block name="Takeoff" strip_button="Takeoff" strip_icon="takeoff.png">
+        <set value="false" var="force_forward"/>
+        <exception cond="stateGetPositionEnu_f()->z > 4.0" deroute="Standby"/>
```
[the remark this answers is not in the document]
Comment (dewagter) [reply conf/flight_plans/tudelft/rotating_wing25kg_EHVB.xml]: sure!

File: conf/airframes/tudelft/rot_wing_25kg.xml [hunk 3]
```diff
@@ -0,0 +1,427 @@
...
+            <define name="ROT_WING_EFF_SCHED_IXX_BODY"              value="0.2845"/>
+            <define name="ROT_WING_EFF_SCHED_IYY_BODY"              value="5.078"/>
+            <define name="ROT_WING_EFF_SCHED_IZZ"                   value="7.051"/>
+            <define name="ROT_WING_EFF_SCHED_IXX_WING"              value="0.712"/>
+            <define name="ROT_WING_EFF_SCHED_IYY_WING"              value="1.77"/>
+            <define name="ROT_WING_EFF_SCHED_M"                     value="25.5"/>
+            <define name="ROT_WING_EFF_SCHED_ROLL_ARM"              value="0.64"/>
+            <define name="ROT_WING_EFF_SCHED_PITCH_ARM"             value="0.556"/>
+            <define name="ROT_WING_EFF_SCHED_HOVER_DF_DPPRZ"        value="0.014007"/>
+            <define name="ROT_WING_EFF_SCHED_HOVER_ROLL_PITCH_COEF" value="{0.64,1.4}"/>
```
[the remark this answers is not in the document]
Comment (dewagter) [reply conf/airframes/tudelft/rot_wing_25kg.xml]: Good point. Especially this section might grow large indeed.

File: conf/flight_plans/tudelft/rotating_wing25kg_EHVB.xml (hunk 1 above)
Comment (dewagter) [reply conf/flight_plans/tudelft/rotating_wing25kg_EHVB.xml]: Tried without and it works.

File: sw/airborne/modules/ctrl/ctrl_eff_sched_rot_wing.c [hunk 4]
```diff
@@ -117,7 +121,7 @@ static abi_event wing_position_ev;
 static void wing_position_cb(uint8_t sender_id UNUSED, struct act_feedback_t *pos_msg, uint8_t num_act)
 {
   for (int i=0; i<num_act; i++){
-    if (pos_msg[i].set.position && (pos_msg[i].idx =  7))
+    if (pos_msg[i].set.position && (pos_msg[i].idx =  SERVO_ROTATION_MECH))
```
Comment (dewagter) [inline sw/airborne/modules/ctrl/ctrl_eff_sched_rot_wing.c:124]: @fvantienen Fixed here.

**Comment by dewagter** [comment]
The current will only work after #3134 is merged, but it is ready to merge.

**Review by gautierhattenberger — APPROVED** [review]

**Pull request merged**